config/lti_dev_key_sample.json

```diff
@@ -25,7 +25,7 @@
     "roles": "$Canvas.membership.roles",
     "isRootAccountAdmin": "$Canvas.user.isRootAccountAdmin"
   },
-  "public_jwk_url": "https://{ccm_app_url}/keys",
-  "target_link_uri": "https://{ccm_app_url}",
-  "oidc_initiation_url": "https://{ccm_app_url}/login"
+  "public_jwk_url": "https://{ccm_app_url}/lti/keys",
+  "target_link_uri": "https://{ccm_app_url}/lti",
+  "oidc_initiation_url": "https://{ccm_app_url}/lti/login"
 }
```

Comment:
@pushyamig, @lsloan, @jonespm, please note the change in configuration here; all the `ltijs` routes start with `/lti` now.

Reply:
Nice improvement on having LTI specific URL with `lti` routes